Chaincase.UI/Pages/SendLabelPage.razor

```diff
@@ -21,4 +24,14 @@
         UiStateService.Title = "SEND";
     }
 
+    public void OnContinue()
+    {
+        StackService.PushStackState(async () =>
+        {
+            await JsRuntime.InvokeVoidAsync("dispatchEventWrapper", "back");
+            NavigationManager.NavigateTo($"send/label");
+        });
+        NavigationManager.NavigateTo("send/amount");
```

Comment:
this code's cool in a view cause it's all presentation & we have no choice here but view-first routing